### ISSUE TITLE
A11y Fail: Date Picker - Need different keyboard command for Help

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1396]
CENG_1509

https://react-magma.cengage.com/version/4.5.0/api/date-pickers/

### Issue:
The Calendar Help modal dialog provides instructions for using various key commands (such as arrows, Enter, Page Down, etc.). One of the key commands is a question mark (?), which can be used to open the Calendar Help modal. This is considered a single printable character key shortcut which can create problems for speech input users and for keyboard users who are prone to accidentally hit keys.

### Acceptance Criteria
- Help panel keyboard shortcut has been removed, so `?` should no longer trigger it
- The shortcut description is removed from the help panel list
- i18n variables and unit tests are cleaned up

### Suggestion:
~Change keyboard shortcut for the keyboard instructions to `ctrl+shift+?`, and update how and where we teach users this shortcut.~

~Edit list of keyboard instructions:~

![image](https://github.com/user-attachments/assets/cb34fcb5-ede1-4459-be64-aeee426f43fe)

~Add to Tooltip on button:~

![image](https://github.com/user-attachments/assets/36b217e0-dafa-42ea-a66b-02cc9b8f7fcc)

**Comment by silvalaura** [comment]
Ensure that we do sufficient cross browser testing when implementing this

**Comment by chris-cedrone-cengage** [comment]
Browsers

**Comment by chris-cedrone-cengage** [comment]
Browsers:
_**Safari 18
Chrome 129
Firefox 131**_

Help panel keyboard shortcut has been removed, so ? should no longer trigger it ✅
The shortcut description is removed from the help panel list ✅
i18n variables and unit tests are cleaned up ✅
